gensim/matutils.py

```diff
@@ -342,6 +342,11 @@ def __iter__(self):
     def __len__(self):
         return self.sparse.shape[1]
 
+    def __getitem__(self, item):
```

Comment:
What is `item`? Row index, column index, something else?

Missing docstring (method semantics, args, returns).